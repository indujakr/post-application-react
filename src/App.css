.App {
  text-align: center;
  background-color: aliceblue;
  min-height: 100vh;
}
