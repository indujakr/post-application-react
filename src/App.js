import "./App.css";
import Dashboard from "./Components/Dashboard/Dashboard";
import Header from './Common/Header'
import Store from "./Store/Store";
function App() {
  return (
    <Store>
      <div className="App">
        <Header/>
        <Dashboard />
      </div>
    </Store>
  );
}

export default App;
