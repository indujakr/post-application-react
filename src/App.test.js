import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from "react-test-renderer";

test('renders learn react link', () => {
  render(<App />);
  const headerElement = screen.getByText(/Post App/i);
  const dashboardElement = screen.getByTestId('post-dashboard')
  expect(headerElement).toBeInTheDocument();
  expect(dashboardElement).toBeInTheDocument();
});

test("match snapshot",()=>{
  const element = renderer.create(<App/>).toJSON();
  expect(element).toMatchSnapshot();
})