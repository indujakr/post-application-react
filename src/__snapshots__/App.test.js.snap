// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`match snapshot 1`] = `
<div
  className="App"
>
  <div
    style={
      Object {
        "background": "#3f51b5",
        "padding": "8px",
      }
    }
  >
    <h1
      style={
        Object {
          "color": "white",
          "margin": "0",
          "textAlign": "left",
        }
      }
    >
      Post App
    </h1>
  </div>
  <div
    className="MuiBox-root MuiBox-root-8 makeStyles-spinner-6"
    data-testid="post-dashboard"
  >
    <div
      className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style={
        Object {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={Object {}}
        />
      </svg>
    </div>
  </div>
</div>
`;
